vsc_pdf_template_editor/lib/src/api_service.dart

```diff
@@ -0,0 +1,12 @@
+import 'dart:core';
+import 'dart:convert';
+import 'consts.dart';
+
+class ApiService {
+
+  Future<Map<String, dynamic>> getDataWidget() async {
```

Comment:
You don't have to do this now, but in a follow-up PR,  this Future can go.  I'm also not sure what `ApiService` means. We won't be calling any APIs.

Reply:
this is used now for loading test widget data. Maybe, I should think of some other class name for this

Reply:
The deserialized json template and data should be injected into the widget via the example app - i.e., the example app performs the loading. But let's address this on a later PR.